CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Fixed Bugs
 
+- [PR #476](https://github.com/konpyutaika/nifikop/pull/476) - **[Operator]** Fixed istio init race condition and pod reconciliation.
+
```

Comment:
Can you also add an entry in `Changed` to indicate that the ZK initContainer was move into the main container.

Reply:
And another one in `Added` to indicate the adding of logic for injected initContainer and container?

Reply:
Done